src/renderer/components/ft-settings-menu/ft-settings-menu.js

```diff
@@ -0,0 +1,16 @@
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+  name: 'FtSettingsMenu',
+  props: {
+    settingsSections: {
+      type: Array,
+      required: true
+    },
+  },
+  methods: {
+    getTitleForSection: function(settingsSection) {
+      return settingsSection.shortTitle !== '' ? settingsSection.shortTitle : settingsSection.title
+    }
```

Comment:
If you handle the fallback in the Settings component instead of here, you won't need any fallback logic here and can get rid of this method, referencing `settingsSection.shortTitle` directly in the template.